package.json

```diff
@@ -58,17 +57,20 @@
     "cz-conventional-changelog": "^3.3.0",
     "eslint": "^8.28.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-import": "^8.5.0",
-    "eslint-plugin-jsdoc": "^8.5.0",
-    "eslint-plugin-no-null": "^8.5.0",
-    "eslint-plugin-prefer-arrow": "^8.5.0",
-    "eslint-plugin-unicorn": "^8.5.0",
+    "eslint-plugin-import": "2.27.5",
```

Comment:
these versions seem to have changed wildly - some up, some down. Intended?

Reply:
eslint migration was using "latest" as the version. I replaced it with the exact version that the package-lock resolved..

Reply:
we can probably keep them as `^` versions, the pinning is what the lock is for.